pkg/morph/client/notary.go

```diff
@@ -516,42 +519,34 @@ func (c *Client) notaryInvoke(committee, invokedByAlpha bool, contract util.Uint
 		Signers: cosigners,
 	}
 
-	// calculate notary fee
-	notaryFee, err := c.client.CalculateNotaryFee(u8n)
+	// define witnesses
+	mainTx.Scripts = c.notaryWitnesses(invokedByAlpha, multiaddrAccount, mainTx)
+
+	mainTx.NetworkFee, err = c.client.CalculateNetworkFee(mainTx)
 	if err != nil {
 		return err
 	}
 
-	// add network fee for cosigners
-	//nolint:staticcheck // waits for neo-go v0.99.3 with notary actors
-	err = c.client.AddNetworkFee(
-		mainTx,
-		notaryFee,
-		c.notaryAccounts(invokedByAlpha, multiaddrAccount)...,
-	)
+	nact, err := notary.NewActor(c.client, []actor.SignerAccount{{
```

Comment:
This actor should be created once per `multiaddrAccount` update (very infrequent), not per-request.